Corona-Warn-App/src/main/res/layout/fragment_rampdown_notice.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Please remove the outer `layout` tag , it is a new fragment we should not use data binding here

Reply:
removed